### PR TITLE
Add Walkie Talkie

### PR DESCRIPTION
**Pull request by vertigoaway**

### CODE REVIEW

**Comment by 24c02** [comment]
hey @vertigoaway!
big fan of the idea of FRS for this, definitely the most plausible-sounding walkie-talkie pitch i've heard so far :3
i will warn you that working with RF is kinda tough, especially for a first project (a lot of it is still dark magic to me :-/)
you'd almost certainly want to go with a prebuilt module for the radio side to avoid going insane (some quick googling points me at the DRA818U, idk about sourcing that, you might have to go non-JLC ¯\\\_(ツ)\_/¯)

tl;dr:
love this idea and it'd be super useful! design would also be some heavy lifting. if you folks are down for the work that entails, reply saying so and i'll merge! :-D

**Comment by vertigoaway** [comment]
haii @24c02 !! :3 
I think we're down for the challenge! I'm familiar with how RF works because I was/is big into ham radio and SDRs ^^
If we have trouble we might make a change to 433MHz and look at the FS1000A and XY-MK-5V for inspiration/guidance.

**Comment by 24c02** [comment]
sweet! godspeed
